src/main/java/seedu/address/model/meeting/Meeting.java

```diff
@@ -68,6 +68,12 @@ public static boolean isValidMeeting(String test) {
         }
     }
 
+    /**
+     * Returns true if {@code meeting} is on the same day.
+     */
+    public boolean isSameDay(Meeting meeting) {
+        return value.substring(0, 6).equals(meeting.value.substring(0, 6));
```

Comment:
I feel that `value.substring(0, 6)` should be abstracted out to another method, `getDay()`.

Reply:
Good point! Done!